build/csslint/xfail.list

```diff
@@ -53,6 +53,7 @@ apps/settings/style/downloads.css 0 3
 apps/sms/style/root.css 0 1
 apps/sms/style/sms.css 0 4
 apps/system/mobile_id/style/mobile_id.css 0 16
+apps/system/style/devtools/devtools_auth.css 1 0
```

Comment:
`object-fit` shouldn't fail, because it's already pretty much standard: http://caniuse.com/#feat=object-fit

My guess is that csslint doesn't know about it yet. Please remove the line from `xfail.list` and we'll get the linter fixed, either by bribing it with a comment, or by updating it, or by fixing a potential problem with the runtime used for parsing, depending on the error message you'll get.